src/utils/functions.ts

```diff
@@ -0,0 +1,67 @@
+import { Person } from '../types';
+import { SortColumns } from './SortColumns';
+import { SortOrders } from './SortOrders';
+
+export function getSortOrder(oldSortOrder: SortOrders) {
+  switch (oldSortOrder) {
+    case (SortOrders.DEFAULT):
+      return SortOrders.DESC;
+    case (SortOrders.DESC):
+      return SortOrders.ASC;
+    default:
+      return null;
+  }
+}
+
+export function sortPeopleBy(
+  people: Person[],
+  sortFilter: SortColumns,
+  sortOrder: string,
+): Person[] {
+  const peopleResult = people;
+
+  switch (sortFilter) {
+    case (SortColumns.NAME): {
+      peopleResult.sort((person1: Person, person2: Person) => {
+        return person2[SortColumns.NAME]
+          .localeCompare(person1[SortColumns.NAME]);
+      });
+
+      break;
+    }
+
+    case (SortColumns.SEX): {
+      peopleResult.sort((person1: Person, person2: Person) => {
+        return person2[SortColumns.SEX]
+          .localeCompare(person1[SortColumns.SEX]);
+      });
+
+      break;
+    }
```

Comment:
in order to not write for every case the same sorting method you can combine cases
```suggestion
  switch (sortFilter) {
    case (SortColumns.NAME): 
    case (SortColumns.SEX): {
      peopleResult.sort((person1: Person, person2: Person) => {
        return person2[sortField]
          .localeCompare(person1[sortField]);
      });

      break;
    }
```